kafka-config.yml.sample

```diff
@@ -0,0 +1,100 @@
+
+      # If several producers/consumers are on the same host an agent can be installed on that host and the
+      # "default_jmx_host" and "default_jmx_port" field can be set once and used for all producers/consumers that 
+      # do not have the "host" or "port" field repsectively.
+      # These fields can be removed if each producer/consumer has it's own "host" and/or "port" field filled out.
+      default_jmx_host: <Default Host to collect JMX Metrics>
+      default_jmx_port: <Default Port to collect JMX Metrics>
+
+      # This field is used to toggle the collection of broker and topic metrics. This is on by default and should only be set to "false"
+      # for the specific case where only producers/consumers are being monitored and "topic_mode" is set to "All".
+      # Any other case this field can be omitted.
+      collect_broker_topic_data: <true or false>
+
+      # Below are the fields used to fine tune/toggle topic metric collection.
+      # In order to collect topics the "topic_mode" field must be set to "All" or "List". If the field is set to "All"
+      # A Zookeeper connection is required, at least the "zookeeper_hosts" field is required, as topics are looked up via Zookeeper.
+      #
+      # It is recommended to use the "List" option to monitor a specific set of topics. If using "List" mode the "topic_list"
+      # field should be filled out. The "topic_list" is a JSON array of topic names to be monitored.
+      # Example of topic_list: '["topic1", "topic2"]'
+      #
+      # If monitoring topics via the "All" or "List" option for "topic_mode", the topic size can be collected from zookeeper by setting 
+      # "collect_topic_size" to true. This operation is intensive and can take a while to collect for a larger number of topics. 
+      # It is recommended to only enable this feature if using a small "topic_list".
+      # If the field is omitted it will default to false.
+      topic_mode: <All, None, or List. All mode requires a zookeeper_host to be specified>
+      topic_list: <JSON Array of Topics to monitor. Ignored if topic_mode is not List>
+      collect_topic_size: <true or false. Indicate if topic size should be collected as it is a very resource intensive metric to collect>
+    labels:
+      env: production
+      role: kafka
+
+  - name: kafka-inventory
+    command: inventory
+    arguments:
+
+      # More information about the zookeeper connection can be found above
+      zookeeper_hosts: <JSON Array of Zookeeper Hosts of the form '[{"host": "localhost", "port": 2181}]'>
+      zookeeper_auth_scheme: <Auth scheme for Zookeeper>
+      zookeeper_auth_secret: <Auth string for Zookeeper>
+      zookeeper_path: <Root path of Kafka nodes in Zookeeper>
+
+      # Below are the fields used to fine tune/toggle topic inventory collection.
+      # In order to collect topics the "topic_mode" field must be set to "All" or "List". If the field is set to "All"
```

Comment:
I think something is missing on the last phrase.

Reply:
Not sure I follow, it continues on the next line. We broke it up for readability. The whole line should read 

```
If the field is set to "All" a Zookeeper connection is required, at least the "zookeeper_hosts" field is required, as topics are looked up via Zookeeper.
```

Reply:
Ok, but next line is empty. Maybe are you referring to line 64?

Reply:
Might be missing something due to how GitHub show comments. So line 64 is a continuation of the end of line 63. Line 65 is blank just to signal a new paragraph starting on line 66.

Reply:
Line 63..66 are fine, I'm talking about current line 92 that ends with _If the field is set to "All"_ and the next line 93 is empty. Sorry for the misunderstanding.

Reply:
Yup totally didn't notice that...sorry. Fixed it!